core/src/avm2/playerglobal/compile_swc.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Is this script a leftover? If so, be sure to update the comment in `core/src/avm2/globals.rs` to not mention it.